libvirt_integration_test.go

```diff
@@ -40,6 +41,26 @@ func TestDisconnectIntegration(t *testing.T) {
 	}
 }
 
+func TestXMLIntegration(t *testing.T) {
+	l := New(testConn(t))
+
+	if err := l.Connect(); err != nil {
+		t.Error(err)
+	}
+	defer l.Disconnect()
+
+	var flags DomainXMLFlags
+	data, err := l.XML("test", flags)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var v interface{}
```

Comment:
Exactly. Libvirt changes the XML definition a bit once the domain has been defined. Otherwise it would be useful to just compare against `test-domain.xml`.